pkg/chart/models/chart.go

```diff
@@ -30,15 +30,16 @@ type Repo struct {
 	Namespace string `json:"namespace"`
 	Name      string `json:"name"`
 	URL       string `json:"url"`
+	Type      string `json:"type"`
 }
 
 // RepoInternal holds the App repository details including auth and checksum
 type RepoInternal struct {
 	Namespace           string `json:"namespace"`
 	Name                string `json:"name"`
 	URL                 string `json:"url"`
+	Type                string `json:"type"`
 	AuthorizationHeader string `bson:"-"`
-	Checksum            string
```

Comment:
Just need to update the comment above, given that the checksum is no longer held here.